lib/ncr/organization.rb

```diff
@@ -1,69 +1,37 @@
 module Ncr
   class Organization
-    BY_CODE = {} # populated later
-    OOL_CODES = Set.new # populated later
-    WHSC_CODE = 'P1122021'
+    WHSC_CODE = "P1122021"
+    OOL_CODES = [
```

Comment:
not that these are added as an array here rather than via the yaml file. That yaml file was added for this tracker story: https://www.pivotaltracker.com/n/projects/1149728/stories/100956358 -- but those org codes all appear to be in the main csv of organizations now. perhaps @phirefly can clarify whether we need to treat OOL orgs differently? (the only difference of this PR is we are no longer using the names in `config/data/ncr/ool_org_codes.yml` but are rather using the default names